### ISSUE TITLE
Install AODN Portal with GeoNetwork with "Parameter" and "Platform" missing

### ISSUE THREAD

**Issue by qingyezeng** [aodn/aodn-portal#2327]
We are trying to set up our own aodn portal. I built aodn-portal master and aodn/geonetwork 2.10.x-imos, and both are installed under Tomcat. Everthing is good, except "Parameter" and "Platform" information don't appear.

![screenshot_2016-09-15_15-20-40](https://cloud.githubusercontent.com/assets/502763/18564499/b6067ff2-7b59-11e6-8450-2878bf1a22bc.png)

I harvested data from https://catalogue-imos.aodn.org.au/geonetwork, also by import/export to get a couple of metadata, the problem remains. Should I apply any more configuration on GeoNetwork? Please give me some clues to spot the problem. 

The link to jekins server of pre-built GeoNetwork war is broken, is it still under maintenance?  Thank you!

**Comment by lwgordonimos** [comment]
@qingyezeng Hi, apologies for the delay in responding! Have you performed the following steps with the vocabulary files? https://github.com/aodn/aodn-portal/wiki/Getting-Started

Regarding the Geonetwork WAR, the document is slightly out of date, so I'll get that updated. In the meantime, the pre-built binaries may now be found here: http://imos-binary.s3-website-ap-southeast-2.amazonaws.com/?prefix=jobs/geonetwork_prod/

**Comment by qingyezeng** [comment]
Thank you for response. I followed the Gettings-Started instruction, both IMOS vocabularies are there and added all the templates including mcp ones. I'm using the latest GeoNetwork war you just provided, but the problem is still there. I suspect I just need more configuration, but I don't have idea where to go. There are a lot options in portal.groovy, except the geonetwork.url, are there any other need to adjust or any more software need to be installed.

I have tried to import some metadata, or harvested from IMOS geonetwork.

Also, is it possible to have a login module to connect geonetwork? like LDAP, so that people can login to get some private metadata into  AODN Portal? 

Is it easy to switch to GeoNetWork 3? I tried to follow the instructions on GeoNetwork 3, but failed. It seems there is new service called "xml.search.imos". Are there any plan to do it?

**Comment by pblain** [comment]
@qingyezeng - sorry for the _**VERY**_ slow response.  Your question got lost somewhere along the line.  I'm not sure if your still perusing this, but here are answers to your questions anyway :  

Unfortunately the AODN portal does not yet support GeoNetwork 3.  We don't support it yet because we depend on the Marine Community Profile (MCP) extension to ISO-19115, which is not supported in GeoNetwork3.  We plan to move to ISO19115-1, which will eliminate this dependency.  But until then we're stuck with GN2.10

Unfortunately there is no security module.  We used to have one but we removed it a long time ago.  All AODN data is public and it's our policy to keep it that way.

Let me know if you have any further questions.  I'll try to answer them more quickly next time!
